python_docs_theme/static/pydoctheme_dark.css

```diff
@@ -124,3 +124,12 @@ div.note {
 img.invert-in-dark-mode {
     filter: invert(1) hue-rotate(.5turn);
 }
+
+/* -- object description styles --------------------------------------------- */
+
+/* C++ specific styling */
+
+.sig.c   .k, .sig.c   .kt,
+.sig.cpp .k, .sig.cpp .kt {
+    color: #5283ff;
+}
```

Comment:
Since the goal is fixing contrast, I would mention it in a comment so that whoever edits the theme colors next is aware of it.